src/app/recipe-list/store/actions/recipe.actions.ts

```diff
@@ -0,0 +1,76 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { createAction, props } from '@ngrx/store';
+import { Recipe } from 'src/models/api.model';
+
+// create recipe
+
+export const addRecipe = createAction('[Recipe List] Add Recipe');
```

Comment:
dobrze jest znać syntax createAction, ale poczytaj o createActionGroup, to jest dość nowe API więc pewnie w kursie tego nie było, a jest fajne bo cały ten plik się skróci do max 20 linijek